packages/api-plugin-carts/src/schemas/cart.graphql

```diff
@@ -595,3 +610,18 @@ extend type Mutation {
     input: UpdateCartItemsQuantityInput!
   ): UpdateCartItemsQuantityPayload!
 }
+
+####################
+ # Subscriptions
+ ####################
+extend type Subscription {
+  "Subscribe to changes to a account cart"
+  accountCartUpdate(
+    input: AccountCartUpdateInput!
```

Comment:
Generally the current pattern is to use `input` for mutations but queries just have arguments at the top level. Since these are more like queries than mutations, I would lean toward removing the extra `input` layer. They would match the arguments for `anonymousCartByCartId` and `accountCartByAccountId` queries.